### PR TITLE
added tests for getting started tutorial using selenium

### PR DESCRIPTION
**Pull request by beliaev-maksim**

### CODE REVIEW

File: tests-bundle/1.7/test_1dot7.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ async def test_deploy(ops_test: OpsTest, lightkube_client, deploy_cmd):
         'kfp-viz',
         # 'knative-eventing', # this is expected to wait for config
         'knative-operator',
-        'knative-serving',  # this is expected to wait for config
+        # 'knative-serving', # this is expected to wait for config
         'kserve-controller',
```
Comment (i-chvets) [inline tests-bundle/1.7/test_1dot7.py:45]: Why is it commented out?

File: tests-bundle/1.7/test_1dot7.py [hunk 2]
```diff
@@ -41,7 +41,7 @@ async def test_deploy(ops_test: OpsTest, lightkube_client, deploy_cmd):
         'kfp-viz',
         # 'knative-eventing', # this is expected to wait for config
         'knative-operator',
-        'knative-serving',  # this is expected to wait for config
+        # 'knative-serving', # this is expected to wait for config
```
Comment (i-chvets) [inline tests-bundle/1.7/test_1dot7.py:44]: Why is it commented out?

File: tests-bundle/1.7/test_tutorial.py [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+from selenium.webdriver.support import expected_conditions
+from selenium.webdriver.support.wait import WebDriverWait
+
+from selenium.webdriver.common.action_chains import ActionChains
+from selenium.webdriver.common.keys import Keys
+
+TESTS_DIR = Path(__file__).resolve().parent
+
+
+class TestGetStartedTutorial:
```
Comment (i-chvets) [inline tests-bundle/1.7/test_tutorial.py:15]: Doc string.

File: tests-bundle/1.7/test_1dot7.py (hunk 2 above)
Comment (beliaev-maksim) [reply tests-bundle/1.7/test_1dot7.py:44]: there are some apps that cause troubles in CI before we run config in DEX in OIDC

File: tests-bundle/1.7/test_tutorial.py (hunk 3 above)
Comment (beliaev-maksim) [reply tests-bundle/1.7/test_tutorial.py:15]: done

**Review by i-chvets — APPROVED** [review]
Approved.

**Pull request merged**